static/app/utils/useCommitters.spec.tsx

```diff
@@ -1,18 +1,23 @@
+import {QueryClient, QueryClientProvider} from '@tanstack/react-query';
+
 import {reactHooks} from 'sentry-test/reactTestingLibrary';
 
-import CommitterStore from 'sentry/stores/committerStore';
 import useCommitters from 'sentry/utils/useCommitters';
 import {OrganizationContext} from 'sentry/views/organizationContext';
 
 describe('useCommitters hook', function () {
   const organization = TestStubs.Organization();
-  const wrapper = ({children}: {children?: React.ReactNode}) => (
-    <OrganizationContext.Provider value={organization}>
-      {children}
-    </OrganizationContext.Provider>
-  );
   const project = TestStubs.Project();
   const event = TestStubs.Event();
+  const queryClient = new QueryClient();
+
+  const wrapper = ({children}: {children?: React.ReactNode}) => (
```

Comment:
TBH I don't think we even need a test for this file anymore, it's basically just testing that `useQuery` works